graph/src/data/subscription/error.rs

```diff
@@ -7,6 +7,8 @@ use prelude::QueryExecutionError;
 pub enum SubscriptionError {
     #[fail(display = "GraphQL error: {}", _0)]
     GraphQLError(QueryExecutionError),
+    #[fail(display = "GraphQL errors: {:?}", _0)]
+    GraphQLErrorList(Vec<QueryExecutionError>),
```

Comment:
This can be refactored into a single variant `GraphQLError(Vec<QueryExecutionError>)`.